internal/utils/templates/init_config.test.toml

```diff
@@ -26,11 +26,21 @@ major_version = 15
 
 [db.pooler]
 enabled = true
+# Port to use for the local connection pooler.
 port = 54329
+# Specifies when a server connection can be reused by other clients.
+# Configure one of the supported pooler modes: `transaction`, `session`.
 pool_mode = "transaction"
+# How many server connections to allow per user/database pair.
 default_pool_size = 20
+# Maximum number of client connections allowed.
 max_client_conn = 100
 
+[realtime]
+enabled = true
+# Bind realtime via either IPv4 or IPv6. (default: IPv6)
+ip_version = "IPv6"
```

Comment:
How about only supporting IPv4 for now? Unless a user requests IPv6, this violates "don't make me think"

Reply:
I think it's better to have explicit configs. Alternatively we can comment out this line if it reduces user's cognitive load.

Reply:
There's a reason we're not exposing all configs available in Realtime, GoTrue etc. - this seems like something that's unnecessary to configure unless a user's machine doesn't support IPv4.

Reply:
As mentioned in the linked issue, realtime defaults to ipv6. User is requesting to use ipv4 because their computer doesn't support ipv6.

Reply:
Yes, what I mean is let’s hardcode it to use IPv4, and if it breaks some setup and an IPv6 option is needed, then we make it configurable

Reply:
I think commenting out the default config is a better option rather than changing the default and risking breaking setups. Also once users rely on IPv4 as the default, it will be much harder to plan an upgrade path as that's potentially another breaking change. 

The concern that the config is irrelevant also applies to other multitenant services. Given that the cli not only serves platform users, but also self hosted, it's inevitable that we need to support a wider range of configs than we previously planned.